src/gwt/src/org/rstudio/studio/client/workbench/prefs/model/UserPrefsAccessorConstants_en.properties

```diff
@@ -977,4 +977,8 @@ checkNullExternalPointersDescription = When enabled, RStudio will detect R objec
 uiLanguageTitle = User Interface Language
 uiLanguageDescription = The IDE's user-interface language.
 
+# Whether RStudio Desktop will use operating system File and Message dialog boxes.
+nativeFileDialogsTitle = Use operating system File and Message dialog boxes
```

Comment:
Label tweak: Use native file and message dialogs? (I don't think capitalization is necessary here)

Reply:
Agree, not Sure why i Capitalized thOse.